connect/runtime/src/test/resources/log4j.properties

```diff
@@ -14,11 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ##
-log4j.rootLogger=OFF, stdout
+log4j.rootLogger=WARN, stdout
```

Comment:
This file changes can be omitted, although it'd be nice to consider whether we want to enable a few log messages in Connect integration tests. This is something that manually needs to be enabled when someone debugs a failure at an integration test.

Reply:
we should enable this for sure, at least at some level. they get stored in truncated logs if successful, and (as of recently) we store full logs for debugging purposes if the test fails. tbh, still not sure how we ended up in a log-free state by default -- it just results in having to turn this on manually, and if test failures are not reproducible locally and only in jenkins, this is a huge pain.

in fact, i'd happily merge this as a separate PR even before this PR :) aside from performance or excessive logging, for tests there isn't much downside in ratcheting up the log level

Reply:
Totally agree too. This is indeed weird given that the two commits that refer to this file, don't show evidence of when this change happened. 

I'll submit another PR that can backported. Thanks!

Reply:
https://github.com/apache/kafka/pull/6745

Reply:
The above PR is merged. I removed this temp commit. Thanks for reviewing!